mods/d2k/rules/defaults.yaml

```diff
@@ -331,3 +332,4 @@
 		Range: 2c0, 5c0
 	ScriptTriggers:
 	WithMakeAnimation:
+
```

Comment:
The ```--upgrade-mod``` command produces those all the time. If empty lines at the end of files are unwanted, the upgrade code should be tweaked accordingly, if possible.